### PR TITLE
fix(condo):DOMA-3405 fixed period select in billing

### PR DESCRIPTION
**Pull request by abshnko**
now any month can be selected, including newer ones

Screenshot 2022-12-23 at 23 25 08

### CODE REVIEW

File: apps/condo/domains/billing/components/BillingPageContent/ReceiptsTable.tsx [hunk 1]
```diff
@@ -13,34 +14,41 @@ import { Table, DEFAULT_PAGE_SIZE } from '@condo/domains/common/components/Table
...
 import { ServicesModal } from '@condo/domains/billing/components/ServicesModal'
 import { useReceiptTableFilters } from '@condo/domains/billing/hooks/useReceiptTableFilters'
+import DatePicker from '@condo/domains/common/components/Pickers/DatePicker'
+import { updateQuery } from '@condo/domains/common/utils/helpers'
+import { getFiltersFromQuery } from '@condo/domains/common/utils/helpers'
+import { getFiltersQueryData } from '@condo/domains/common/utils/filters.utils'
 
 
 const SORTABLE_PROPERTIES = ['toPay']
+const INPUT_STYLE = { width: '18em' }
```
Comment (SavelevMatthew) [inline apps/condo/domains/billing/components/BillingPageContent/ReceiptsTable.tsx:29]: Why in em?

File: apps/condo/domains/billing/components/BillingPageContent/ReceiptsTable.tsx [hunk 2]
```diff
@@ -72,6 +79,20 @@ export const ReceiptsTable: React.FC<IContextProps> = ({ context }) => {
         setModalIsVisible(false)
     }
 
+    const periodMetaSelect = useMemo(() => {
```
Comment (SavelevMatthew) [inline apps/condo/domains/billing/components/BillingPageContent/ReceiptsTable.tsx:82]: meta?

File: apps/condo/domains/billing/components/BillingPageContent/ReceiptsTable.tsx [hunk 3]
```diff
@@ -72,6 +79,20 @@ export const ReceiptsTable: React.FC<IContextProps> = ({ context }) => {
...
+        return (
+            <Space direction='vertical' size={12} style={{ marginLeft:'18px' }}>
```
Comment (SavelevMatthew) [inline apps/condo/domains/billing/components/BillingPageContent/ReceiptsTable.tsx:84]: why margin?

Comment (SavelevMatthew) [reply apps/condo/domains/billing/components/BillingPageContent/ReceiptsTable.tsx:84]: wrap + size={[12, 18]} should work

**Review by toplenboren — APPROVED** [review]

**Pull request merged**